com.microsoft.mrtk.spatialmanipulation/Solvers/SolverHandler.cs

```diff
@@ -578,8 +580,9 @@ private bool IsInvalidTracking()
                 bool trackingLeft = IsHandTracked(Handedness.Left);
                 bool trackingRight = IsHandTracked(Handedness.Right);
 
-                return (currentTrackedHandedness == Handedness.Left && !trackingLeft) ||
-                       (currentTrackedHandedness == Handedness.Right && !trackingRight);
+                return
+                    (trackingLeft && currentTrackedHandedness.IsMatch(Handedness.Left)) ||
+                    (trackingRight && currentTrackedHandedness.IsMatch(Handedness.Right));
```

Comment:
what is the reasoning for the change to the logic here? (the !trackingLeft to trackingLeft part)

Reply:
Yeah good catch :)   Just caught that myself via testing...

Oddly enough...the code works in most cases :)